include/secp256k1_schnorr_adaptor.h

```diff
@@ -57,7 +57,7 @@ SECP256K1_API const secp256k1_adaptor_nonce_function_hardened secp256k1_nonce_fu
  *  Out:   presig65: pointer to a 65-byte array to store the adaptor signature.
  *  In:       msg32: the 32-byte message being signed.
  *          keypair: pointer to an initialized keypair.
- *        adaptor33: pointer to a 33-byte compressed adaptor point.
+ *        adaptor: pointer to an adaptor point.
```

Comment:
nit: 344620e renamed `adaptor` to `adaptor33`. Just wanna make sure that this revert is intentional.